CONTRIBUTING.md

```diff
@@ -50,6 +50,13 @@ We can't guarantee that we'll accept pull requests and may ask you to make some
 Occasionally, we might also have logistical, commercial, or legal reasons why we can't accept your work but we'll try to find an alternative way for you to contribute in that case.
 Remember that many community members have become regular contributors and some are now even Neo employees!
 
+## Go driver extra contribution guidelines
+
+Make sure to install the pre-commit hook to your local repository:
+
+```shell
+cp hooks/pre-commit .git/hooks/
```

Comment:
```suggestion
ln -s ../../hooks/pre-commit .git/hooks/pre-commit
```
might be another option in case the hooks gets updated at some point.

Reply:
will not work well on Windows afair

Reply:
true, will cp work?

Reply:
Another interesting find I just made https://stackoverflow.com/a/57592188/2376882

Reply:
> will cp work?
Git Bash and friends support it well